app-creator/client/src/widgets/template-wizard/test/template-wizard_test.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ *  @fileoverview This file tests the template-wizard widget.
+ */
+import { TemplateWizard } from '../template-wizard';
+import { fixture, html, expect, assert } from '@open-wc/testing';
+
+suite('templates-tab', () => {
```

Comment:
There are a few more tests that are useful here. For example, that the continue button is disabled while certain fields are blank, and whether the backgroundColor/color style logic is applied correctly to a template. I would recommend adding tests for any logic for which you need to guarantee something to be true for the rest of the app creator to work correctly, particularly around the template JSON.

Reply:
Agreed that we should add unit tests to make sure it works properly :)